docs/resources/openid_client.md

```diff
@@ -53,7 +53,7 @@ resource "keycloak_openid_client" "openid_client" {
       URIs for security. This client should be used for applications using the Implicit grant flow.
   - `BEARER-ONLY` - Used for services that never initiate a login. This client will only allow bearer token requests.
 - `client_secret` - (Optional) The secret for clients with an `access_type` of `CONFIDENTIAL` or `BEARER-ONLY`. This value is sensitive and should be treated with the same care as a password. If omitted, this will be generated by Keycloak.
-- `client_authenticator_type` - (Optional) Defaults to `client-secret` The authenticator type for clients with an `access_type` of `CONFIDENTIAL` or `BEARER-ONLY`. Can be one of the following:
+- `client_authenticator_type` - (Optional) Defaults to `client-secret`. The authenticator type for clients with an `access_type` of `CONFIDENTIAL` or `BEARER-ONLY`. The following types are available on an unmodified Keycloak server:
```

Comment:
Would like to document this in a consistent way if possible.

Are there other parameters which aren't validated because developers can extend the server?